runbot_coverage/models/runbot_repo.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import sys
+from openerp import api, fields, models
+
+
+class RunbotRepo(models.Model):
+    _inherit = 'runbot.repo'
+
+    use_coverage = fields.Boolean('Coverage', default=False)
+    coverage_config = fields.Char(
+        'Configuration file',
+        help='Your coveragerc relative to the build path.')
+    coverage_command = fields.Char(
```

Comment:
yes, the external dependency must go away. The whole dance is necessary for two reasons: On debian, there's no `coverage`, but the thing is called `python-coverage`. So we just can't use the dependencies mechanism. Then second in cases with a weird deployment, you want to use some specific non-system version of the script for isolation reasons. In my case the latter would be pointing to some version of coverage in runbot's home